release_util/management/commands/__init__.py

```diff
@@ -214,7 +214,7 @@ def __init__(self, stderr, database_name, migrations=None):
 
         # Regex built to match migration output like this line:
         #    Applying release_util.0001_initial...  OK
-        self.migration_regex = re.compile(r'Applying (?P<app_name>[^\.]+)\.(?P<migration_name>[^\.]+)[\. ]+(?P<success>[OK]*)$')
+        self.migration_regex = re.compile(r'Applying (?P<app_name>[^.]+)\.(?P<migration_name>[^.]+)[. ]+(?P<success>(OK)?)$')
```

Comment:
nit: Do we have a comment anywhere that shows some example output that we are attempting to parse? If not could you please add it?